templates/frontend/objects/article_details.tpl

```diff
@@ -117,7 +117,7 @@
 									{/if}
 								</span>
 							{/if}
-							{if $author->getData('orcid')}
+                            {if $author->getData('orcid') && $author->getData('orcidAccessToken')}
```

Comment:
If we go this route, let's make a helper function in the `Author` class to encapsulate the details of authentication. Something like `$author->getOrcid()` or `$author->getAuthenticatedOrcid()`.

Reply:
Thanks @NateWr . Will do a helper function for that if we add that.

I was thinking, until we have a proper solution, may be only display the green  icon ,  so that it is clear. What do you think ?

https://github.com/withanage/ojs/blob/stable-3_3_0/templates/frontend/objects/article_details.tpl#L122

Reply:
I think any change to 3.3 needs to be discussed with a larger group. I don't feel I know well enough how this will impact our community. If you think it's important, can you file an issue? Or go back to the PS team and ask them to do so. Then we can try to gather feedback from a wider group of users.